src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlCreation.cs

```diff
@@ -24,18 +25,31 @@ public partial class TestWorkspace
             string[] metadataReferences = null,
             string extension = null,
             bool commonReferences = true,
-            bool isMarkup = true)
+            bool isMarkup = true,
+            string[] fileContainingFolders = null)
```

Comment:
This might need a bit of comment explaining what this is -- so it's an array with one entry for each document, but then you only have a single folder name there? Or are you doing a string.Split somewhere?